app/models/piece.rb

```diff
@@ -6,10 +6,23 @@ def self.types
     %w[King Queen Rook Bishop Knight Pawn]
   end
 
-  def self.colors
+  def self.color
     %w[White Black]
   end
 
+  def move_to!(piece, destination_x, destination_y)
+    destination_piece_check = Piece.find_by(column_coordinate: destination_x, row_coordinate: destination_y).present?
```

Comment:
We tried using .exists?, however we get this error. Also tried .exist?, and got the same 

Piece#move_to! when a piece with the opposite color is at the destination coordinates the other piece is removed from the board
     Failure/Error: destination_piece_check = Piece.find_by(column_coordinate: destination_x, row_coordinate: destination_y).exists?
     
     NoMethodError:
       undefined method `exists?' for #<Piece:0x007f0934851c88>